.github/workflows/publish-pyscitt-to-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: "Publish pyscitt package to PyPi"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build_and_publish:
+    name: "Publish pyscitt package to PyPi"
+    runs-on: ubuntu-latest
+
+    if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
+
+    environment:
+      name: pypi
+      url: https://pypi.org/p/pyscitt
+
+    permissions:
+      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: Build package
+        run: |
+          python3 -m pip install build
+          cd pyscitt
+          python3 -m build
+      
+      - name: Publish package to PyPI
```

Comment:
Instead of having it as a separate action just merge it to `publish-gh-release-files.yml` and maybe rename it to reflect that it is an action that does something on tag pushes.

It is basically a duplicate but instead of storing the built files in a github release it pushes it to pypi

Reply:
I created a separate action to make the two jobs run independently and because I didn't want to modify the existing job to create the GitHub release. But indeed they are sharing most of the steps and it can make sense to merge them.

I created a single `release` action with two independent jobs to create a GH release and publish pyscitt to PyPi. Let me know what you think.